app/routes/_gcn.docs.contributing.project/route.mdx

```diff
@@ -0,0 +1,196 @@
+  height="542"
+  className="height-auto"
+/>
+
+We create Issues for all work. All Issues should have a detailed title, description, and acceptance criteria. All issues must be pointed before moving into the Backlog or In Progress. Some Project metadata is also editable from an Issue page.
+
+<img
+  src={issueScreenshot}
+  alt="Screen shot of a GitHub Issue"
+  width="2548"
+  height="1972"
+  className="height-auto"
+/>
+
+## Pointing
+
+We point tickets as a group during planning meetings.
+We use the description and acceptance criteria written in the github Issue to ensure group understanding of the work.
+Once we have agreed that we understand the scope and request, we vote on the level of complexity for that body of work.
+We use the unique numbers in the fibbonacci sequence to represent complexity. We only go up to 21 points, making our options 1, 2, 3, 5, 8, 13, and 21. We also have the additional options of `coffee` or `question mark`.
+
+<Table bordered compact stackedStyle="headers">
+  <thead>
+    <tr>
+      <th scope="col">Complexity</th>
+      <th scope="col">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1</td>
+      <td>Something incredibly straightforward such as a copy fix.</td>
+    </tr>
+    <tr>
+      <td>2</td>
+      <td>A small uncomplicated change</td>
+    </tr>
+    <tr>
+      <td>3</td>
+      <td>The average bugfix or small feature</td>
+    </tr>
+    <tr>
+      <td>5</td>
+      <td>This is a more robust feature or a more complex bug</td>
+    </tr>
+    <tr>
+      <td>8</td>
+      <td>
+        This is quite large. It may need to be broken down into smaller steps
+        where possible.
+      </td>
+    </tr>
+    <tr>
+      <td>13</td>
+      <td>
+        This is where we start deeply questioning. If a ticket is 13 points, it
+        is possible that it is not broken down into small enough steps and would
+        benefit from further breaking down. Or perhapse the acceptance criteria
+        are too ambiguous.
+      </td>
+    </tr>
+    <tr>
+      <td>21</td>
+      <td>
+        This is a wildly difficult task. This will take more than an
+        iteration/sprint, likely more than one.
+      </td>
+    </tr>
+    <tr>
+      <td>coffee</td>
+      <td>Abstaining from voting.</td>
+    </tr>
+    <tr>
+      <td>question mark</td>
+      <td>
+        This is used when the description or acceptance criteria are too vague
+        to make an accurate estimate or there are any outstanding questions that
+        would prevent accurate pointing.
+      </td>
+    </tr>
+  </tbody>
+</Table>
+
+Tickets with clear descriptions and acceptance criteria are pointed before being placed in the backlog.
+
+## Iterations/sprints
+
+We use both sprint and iteration to mean a two week period of work. Our github Project board uses iterations though these spans of work are more frequently known as sprints. Our team uses both words interchangeably.
+
+## Planning Meetings
+
+TBD.
```

Comment:
While this documentation is very useful for our team, I do wonder about putting it in the "Contributing" documentation.  I think it's good to be open about how our team is operating, and this is very useful for new members, but we're not planning to make the planning meeting open.